front/cypress/e2e/idea_form_settings.cy.ts

```diff
@@ -185,6 +186,177 @@ describe('Idea form settings', () => {
     });
   });
 
+  describe('Input form tab', () => {
```

Comment:
I think a test case should be added for a timeline project with one phase which is a native survey, because otherwise we cannot be sure that the "Input form" tabs will be hidden for such a project.